addons/account/models/sequence_mixin.py

```diff
@@ -106,9 +106,8 @@ def _constrains_date_sequence(self):
                 and not record._sequence_matches_date()
             ):
                 raise ValidationError(_(
-                    "The %(date_field)s (%(date)s) doesn't match the sequence number of the related %(model)s (%(sequence)s)\n"
-                    "You will need to clear the %(model)s's %(sequence_field)s to proceed.\n"
```

Comment:
if i'm not mistaken model is not used in the docstring anymore so could be remove, same goes for sequence field. 
Also maybe we could reorder the field to be in order with the docstring :smile: